string name
int32 robot_num
int32 num_geom
rost_viewer_geometry_data[] geom